src/test/components/components-basic.test.ts

```
@@ -0,0 +1,82 @@
+    const testTitles = [
+      "Article de Test",
+      "Un Titre Avec Accents éàü",
+      "Special !@# Characters"
+    ];
+
+    testTitles.forEach((input) => {
+      // Simuler la logique de slug simple
+      const slug = input
+        .toLowerCase()
+        .normalize("NFD")
+        .replace(/[\u0300-\u036f]/g, "") // Supprimer les accents
```

Biome (1.9.4)

[error] A character class cannot match a character and a combining character.

A character and a combining character forms a new character. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)